### ISSUE TITLE
Change DM Phase

### ISSUE THREAD

**Issue by aaronmberger-nwfsc** [pacific-hake/hake-assessment#596]
Thanks to Kelli for pointing out that the Dirichlet Multinomial parameters could be estimated in the last phase (similar to other iterative data weighting approaches) rather than in earlier phases.  Currently it is set to phase 2, which is fairly early.  The phase choice of 2 versus last_phase for DM parameters makes zero difference so not going to worry about this now.  But adding a 2021 milestone to revisit what makes the most sense moving forward for DM parameters.

**Comment by kellijohnson-NOAA** [comment]
- [ ] ensure DM parameter phases are the last phase of the model
- [ ] check phase of added variance for the survey, which maybe should happen after DM phase, not entirely sure, but at least after biological parameters are estimated

**Comment by andrew-edwards** [comment]
Bumped to 2021

**Comment by aaronmberger-nwfsc** [comment]
I checked the DM phasing and the survey extra SD phasing and the estimation phase made no difference in model results.  So I went ahead and moved these both to the final phase based on general principle (as stated previously).  I also added a "bridging" model that makes this change.  I did this just so that when moving forward this year with model development we retain these changes.  I don't see a need to show this as a bridging step in the document I don't think.  As this is now done and changes will be included in 2021 base model development, I will close this now.
